py3status/modules/backlight.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""
+Display the current screen backlight level.
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds (default: 10s)
+    color:  The text color (default: "#FFFFFF")
+    device: The backlight device
+            If not specified the plugin will try to detect it automatically
+
+Format status string parameters:
+    {level} brightness
+
+@author Tjaart van der Walt (github:tjaartvdwalt)
+@license BSD
+"""
+
+from __future__ import division
+
+import os
+from time import time
+
+
+def get_device_path():
+    for (path, devices, files) in os.walk('/sys/class/backlight/'):
+        for device in devices:
+            if 'brightness' in os.listdir(path + device) and \
+               'max_brightness' in os.listdir(path + device):
+                return path + device
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 10
+    color = '#FFFFFF'
```

Comment:
You use `self.device` but it is not defined.  All config options should have a default even if it is None.

Actually having looked at things it might be good if you find a device if none is defined.  On my system I have two `acpi_video0` and `intel_backlight` it would work with either they are both the same device.  I've not tried with multiple monitors etc